src/store/fenceConnection/slice.ts

```diff
@@ -1,24 +1,28 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { initialState } from 'store/fenceConnection/types';
 import { FENCE_CONNECTION_STATUSES, FENCE_NAMES } from 'common/fenceTypes';
 import { checkFenceAuthStatus, connectToFence, disconnectFromFence } from './thunks';
 
 export const FenceConnectionState: initialState = {
   connectionStatus: {
+    [FENCE_NAMES.dcf]: FENCE_CONNECTION_STATUSES.unknown,
     [FENCE_NAMES.gen3]: FENCE_CONNECTION_STATUSES.unknown,
     [FENCE_NAMES.cavatica]: FENCE_CONNECTION_STATUSES.unknown,
   },
   fencesInfo: {
+    [FENCE_NAMES.dcf]: undefined,
```

Comment:
since it's legacy code let it be but I would have used `null` or `'' `(empty string) instead.